src/main/java/org/folio/rest/persist/QueryHolder.java

```diff
@@ -9,13 +9,15 @@
 public class QueryHolder {
 
   private String table;
+  private String searchField;
   private String query;
   private int offset;
   private int limit;
   private String lang;
 
-  public QueryHolder(String table, String query, int offset, int limit, String lang) {
```

Comment:
You could leave constructor with default behavior to not modify any other classes:
```java
  public QueryHolder(String table, String query, int offset, int limit, String lang) {
    this(table, HelperUtils.JSONB, query, offset, limit, lang);
  }
```